packages/vscode-common/getExtensionApi.ts

```diff
@@ -1,41 +1,42 @@
 import type {
   CommandServerApi,
+  ExcludableSnapshotField,
+  ExtraContext,
+  ExtraSnapshotField,
+  HatTokenMap,
   IDE,
   NormalizedIDE,
   SerializedMarks,
+  SnippetMap,
   TargetPlainObject,
+  TestCaseSnapshot,
   TextEditor,
 } from "@cursorless/common";
-import type {
-  ExcludableSnapshotField,
-  ExtraContext,
-  ExtraSnapshotField,
-  Graph,
-  TestCaseSnapshot,
-  ThatMark,
-} from "@cursorless/cursorless-engine";
 import * as vscode from "vscode";
 import type { SyntaxNode } from "web-tree-sitter";
-import type { SnippetMap } from "../cursorless-engine/snippets/snippet.types";
-import type { Target } from "../cursorless-engine/typings/target.types";
 
-interface TestHelpers {
-  graph: Graph;
+export interface TestHelpers {
   ide: NormalizedIDE;
   injectIde: (ide: IDE) => void;
 
-  // FIXME: Remove this once we have a better way to get this function
-  // accessible from our tests
-  plainObjectToTarget(
+  hatTokenMap: HatTokenMap;
+
+  commandServerApi: CommandServerApi;
+
+  toVscodeEditor(editor: TextEditor): vscode.TextEditor;
```

Comment:
this is needed by [`groupByDocument.test.ts`](https://github.com/cursorless-dev/cursorless/pull/1280/files#diff-1b735f813e740060570a879c9bad8493961c73dd1a618a9f9934ba9f88020f0a).  I think it's better to pass it out here rather than test harness directly importing it, as uses dangerous implementation details